test-runner/src/main/scala/com/typesafe/spark/test/mesos/MesosIntTestHelper.scala

```diff
@@ -25,21 +25,21 @@ trait MesosIntTestHelper { self: FunSuite =>
    * Creates a SparkContext based on the given properties and reports test result to runner
    * @param name name of the job and mesos framework. The SPARK_FRAMEWORK_PREFIX-$name is used as the Spark
    *             application name.
-   * @param ps  key-value pairs of Spark configuration
+   * @param properties  key-value pairs of Spark configuration
    * @param t function that contains the testcase
    * @return ()
    */
-  def runSparkTest(name: String, ps: (String, String)*)(t: (SparkContext) => Unit) {
+  def runSparkTest(name: String, properties: () => Seq[(String, String)], tags: Seq[Tag] = Nil)(t: (SparkContext) => Unit) {
```

Comment:
how about call by name idiomatic syntax: properties:  => Seq[(String, String)]  

Then that means:
  runSparkTest("simple count in coarse-grained mode with role",
     List("spark.mesos.coarse" -> "true", "spark.mesos.role" -> cfg.role),
    List(Tag("skip-dcos"))) { sc =>
    testRole(sc, true)
  }